### PR TITLE
Update : Allen

### PR DESCRIPTION
**Pull request by AllenP3**
added AllenP's card

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4739,6 +4739,31 @@ <h2>A project for first-time contributions</h2>
...
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://www.youtube.com/@NeuralNine" target="_blank" title="First Resource">NeuralNine</a>
+              </li>
+              <li>
+                <a href="#" target="https://www.youtube.com/@freecodecamp" title="Second Resource">Freecodecamp</a>
+              </li>
+              <li>
+                <a href="#" target="https://www.w3schools.com/" title="Third resource">W3schools</a>
```
Comment (TenTraicion) [inline index.html:4761]:
```suggestion
                <a href="https://www.youtube.com/@freecodecamp" target="_blank" title="Second Resource">Freecodecamp</a>
              </li>
              <li>
                <a href="https://www.w3schools.com/" target="_blank" title="Third resource">W3schools</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@AllenP3, fix your card by following the given suggestions. Thanks!

**Comment by CandidDeer** [comment]
@AllenP3 Please note that we will be closing this PR 7 days after the initial opening if the requested changes have not been resolved by then. Please always feel free to reach out if you have any questions on how to resolve. Thx.

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@AllenP3, Congratulation on your PR Merge, and Thanks for your Contribution. You can check your card in our [Live Website](https://syknapse.github.io/Contribute-To-This-Project/)! It may take 2-3 minutes to make your card visible in our website. 

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**